Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -59,6 +59,10 @@ void HTMLLinkElement::inserted()
 {
     HTMLElement::inserted();
 
+    if (!document().is_fully_active()) {
+        return;
+    }
```

Comment:
Hmm, I'm not sure this is the right fix.

Isn't the bug that we don't check before calling `fetch_and_process_linked_resource()` that the link element is "browsing-context connected"?

Reply:
yes, I converted to draft because `move-loaded-link-stylesheet-between-documents.txt` regressed after this change. I think we should introduce "browsing context connected" hook and move fetching there, instead of doing that after insertion.

Reply:
actually, `move-loaded-link-stylesheet-between-documents.txt` is not a regression. this test relied on the fact fetching callback will be processed before `println("Sheets in new doc: " + document.styleSheets.length)` which is no longer true. this test does not produce the same output reliably in other browsers either.

I changed the check to look for browsing context instead of checking if document is active. that follows the spec more precisely and probably more correct.